Mlem/Views/Tabs/Inbox/InboxFeedView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  InboxFeedView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-04-01.
+//
+
+import Foundation
+import SwiftUI
+
+struct InboxFeedView<T: TrackerProtocol>: View where T.Item: InboxItem {
+    @ObservedObject var tracker: T
+    
+    var body: some View {
+        if tracker.loadingState == .done, tracker.items.isEmpty {
+            noItemsView()
+        } else {
+            itemsListView()
+        }
+    }
+    
+    @ViewBuilder
+    func itemsListView() -> some View {
+        ForEach(tracker.items, id: \.uid) { item in
+            VStack(spacing: 0) {
+                inboxItemView(item: item.toAnyInboxItem())
+                    .onAppear {
+                        tracker.loadIfThreshold(item)
+                    }
+                
+                Divider()
+            }
+        }
+        
+        EndOfFeedView(loadingState: tracker.loadingState, viewType: .cartoon)
```

Comment:
`EndOfFeedView` contains `LoadingView(whatIsLoading: .posts)` - we should probably add a parameter to `EndOfFeedView` that allows us to change `whatIsLoading`, since `.posts` doesn't make much sense here.